README.md

```diff
@@ -179,6 +179,82 @@ At a glance you can see that the activations of the middle hidden node
 results in predictions of class 0 while the activation of the left-most
 and right-most hiddent nodes result in predictions of class 1.
 
+### Guidance On Graph Customization
+
+You can modify the node activation graph. The node activation graph
+consists of a pyplot graph and a networkx graph. You can change the
+color and marker of the points in the pyplot graph, and attributes like 
+the color and shape of each node in the networkx graph.
+
+For the pyplot graph, you can use method `set_x_color` and `set_x_marker`.
+
+For the networkx graph, first, you need to use the `get_graph` method to get
+the networkx graph from `DenseGraph`. Then, you can modify the graph as needed.
+One possible way is to loop through the nodes of the graph by index and use 
+`set_node_attributes` and `set_edge_attributes` from `networkx`. Finally, you 
+can use the `set_graph` method to pass the modified graph to `DenseGraph` and 
+call `render` to get the visualization.
+
+Below is an example using the XOR function: (assume we already had a keras model; you can get
+one using the codes above)
+
+```python
+    from networkx import set_node_attributes, set_edge_attributes
+    
+    # "model" is already a trained keras model.
+    dg = DenseGraph(model)
+    
+    # Get the networkx graph.
+    the_graph = dg.get_graph()
+    
+    # Loop through the graph.
+    for l in range(len(model.layers)):
```

Comment:
this should iterate over the graph not the model

Reply:
I think the user may need to know the number of layers and the number of nodes in each layer (if they want to set each layer different), so it may be necessary for them to get the length of `model.layers` and the input / output shape of the layer? @gallettilance